modules/dedicated-cluster-install-deploy.adoc

```diff
@@ -60,28 +66,18 @@ If either operator does not appear as installed, to troubleshoot further:
 * Switch to the *Operators* → *Installed Operators* page and inspect
 the *Status* column for any errors or failures.
 * Switch to the *Workloads* → *Pods* page and check the logs in any Pods in the
-`openshift-logging` and `openshift-operators-redhat` projects that are reporting issues.
+`openshift-logging` project that is reporting issues.
 
 . Create a cluster logging instance:
 
-.. Switch to the *Administration* -> *Custom Resource Definitions* page.
-
-.. On the *Custom Resource Definitions* page, click *ClusterLogging*.
-
-.. On the *Custom Resource Definition Overview* page, select *View Instances* from the *Actions* menu.
+.. Switch to the *Operators* → *Installed Operators* page.
 
-.. On the *Cluster Loggings* page, click *Create Cluster Logging*.
-+
-You might have to refresh the page to load the data.
+.. Click the installed *Cluster Logging* Operator.
 
-.. In the YAML, replace the code with the following:
-+
-[NOTE]
-====
-This default cluster logging configuration should support a wide array of environments. Review the topics on tuning and
-configuring the cluster logging components for information on modifications you can make to your cluster logging cluster.
-====
+.. Under the *Overview* tab, click *Create Instance* . You should now copy and
```

Comment:
maybe 
```
.. Under the *Overview* tab, click *Create Instance*. Paste the following YAML definition into the window that displays.
```

>! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as
appropriate.

>! below adj
Do not use to indicate a relative location in a document, as in “the information below.”

Reply:
This is fantastic, thank you.